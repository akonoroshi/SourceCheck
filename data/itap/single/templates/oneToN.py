def oneToN(n):
    text = ''
    for i in range(1, n + 1):
        text += str(i)
    return text
