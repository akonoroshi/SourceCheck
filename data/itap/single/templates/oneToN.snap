Module(list {
    FunctionDef:oneToN(arguments(list {
        arg:n(null)
    }, null, list, list, null, list), list {
        Assign(list {
            Name:text(Store)
        }, Str:)
        For(Name:i(Store), Call(Name:range(Load), list {
            Num:1
            BinOp(Name:n(Load), Add, Num:1)
        }, list), list {
            AugAssign(Name:text(Store), Add, Call(Name:str(Load), list {
                Name:i(Load)
            }, list))
        }, list)
        Return(Name:text(Load))
    }, list, null)
})